packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -299,22 +341,25 @@ class Metrics {
     return null;
   }
 
+  // TODO: we can add clientName to userAgent for CA to parse and add a config value
```

Comment:
Do we have a JIRA for this TODO?

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-122143